src/modules/zenoh/module.yaml

```diff
@@ -6,9 +6,12 @@ parameters:
 
         ZENOH_ENABLE:
             description:
-                short: Zenoh Enable
-                long: Zenoh
+                short: Enable Zenoh
+                long: |
+                    Set true (1) to start the Zenoh driver module (a.k.a the "Zenoh-Pico Node").
+                    See https://docs.px4.io/main/en/middleware/zenoh and
+                    https://docs.px4.io/main/en/modules/modules_driver.html#zenoh
             category: System
-            type: int32
+            type: boolean
```

Comment:
This was a bit messed up. I think this is the right way to do a boolean - and you should use either a boolean or an enum when you only want particular values.

Reply:
Hm interesting I though PX4 always only had float and int32, didn't knew boolean worked.
So yeah boolean is way better.

Reply:
Great. Enum works too if you think there might be other configuration you want to do through the param later.

It gets [converted into an INT32](https://github.com/PX4/PX4-Autopilot/blob/main/Tools/module_config/generate_params.py#L120-L122) by the toolchain